assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -157,6 +157,8 @@ che.docker.cpuset_cpus=NULL
 che.docker.cpu_period=0
 che.docker.cpu_quota=0
 
+# Adds options for che user id
+che.docker.che.user.id=
```

Comment:
Please do not use che twice in property name. We also have convention on usage of underscore to separate works of some term and dots to separate structural elements.
What about `che.docker.user_id` ? @TylerJewell since you are managing user-friendly names, please take a look